### PR TITLE
Save user ui theme preference

### LINKED ISSUE #112: Save user UI preference
Save the preference of user for the UI theme

### PR DESCRIPTION
**Pull request by Ma-Yueran**
Make GuiSettings save user's UI theme preference.

Update storage tests.

Fix unchecked type warning in ThemeWindow for ListView.

Resolve #112

### CODE REVIEW

**Comment by Ma-Yueran** [comment]
Also delete UI old classes that are no longer used.

File: src/main/java/seedu/address/commons/core/GuiSettings.java [hunk 1]
```diff
@@ -73,7 +83,8 @@ public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Width : " + windowWidth + "\n");
         sb.append("Height : " + windowHeight + "\n");
-        sb.append("Position : " + windowCoordinates);
+        sb.append("Position : " + windowCoordinates + "\n");
+        // sb.append("Theme: " + uiTheme);
```
Comment (anthony6401) [inline src/main/java/seedu/address/commons/core/GuiSettings.java]: Perhaps you mean to append the themeName here?

**Review by anthony6401 — CHANGES_REQUESTED** [review]
Some minor change that need to be addressed. Other than that, LGTM!

File: src/main/java/seedu/address/commons/core/GuiSettings.java (hunk 1 above)
Comment (Ma-Yueran) [reply src/main/java/seedu/address/commons/core/GuiSettings.java]: Oh, yeah, thank you for pointing that out

**Review by anthony6401 — APPROVED** [review]
LGTM!

**Pull request merged**